bip-0052.mediawiki

```diff
@@ -0,0 +1,302 @@
+<pre>
+  BIP: 52
+  Title: Durable, Low Energy Bitcoin PoW
+  Author: Michael Dubrovsky <mike+bip@powx.org>
+          Bogdan Penkovsky <bogdan+bip@powx.org>
+  Discussions-To: <mike+bip@powx.org>
```

Comment:
Is this a mailing list?

"While a BIP is in private discussions (usually during the initial Draft phase), a Discussions-To header will indicate the mailing list or URL where the BIP is being discussed. No Discussions-To header is necessary if the BIP is being discussed privately with the author, or on the bitcoin email mailing lists."

Reply:
No, it is not.

Reply:
Removed the email from Discussions-To in c689a92

Reply:
Removed the empty field causing syntax error in fecbe1f